pyexif/__init__.py

```diff
@@ -66,12 +66,14 @@ def _runproc(cmd, fpath=None):
 
 
 class ExifEditor(object):
-    def __init__(self, photo=None, save_backup=False):
+    def __init__(self, photo=None, save_backup=False, extra_opt=None):
```

Comment:
Since exiftool has a lot of options, what if this were a list, and then join them together? E.g.:

`def __init__(self, photo=None, save_backup=False, extra_opts=None):`
`    ...`
`    extra_opts = extra_opts or []`
`    if save_ backup:`
`        extra_opts.append("-overwrite_original_in_place")`
`    self._optExpr = " ".join(extra_opts)`

This way someone who might want to pass several options can do it more pythonically.